src/ui/panels/quests.js

```diff
@@ -188,27 +196,29 @@ export class Quests {
             for (var enemy in enemies) {
                 let enemiesKilled = characterData.getEnemiesKilled(level, enemy);
 
-                for (
-                    var tier = 1;
-                    tier <=
-                    characterData.calcQuestTier(enemiesKilled, scene.questAmounts[enemy]);
-                    tier++
-                ) {
+                let questTier = characterData.calcQuestTier(
+                    enemiesKilled,
+                    scene.questAmounts[enemy]
+                );
+
+                for (var tier = 1; tier <= questTier; tier++) {
```

Comment:
I know it was already there but `var` -> `let`

Reply:
https://github.com/EthanVieira/rs-clicker/pull/67/commits/61dfc063b2991372be5e92829d7e8ae2813cf74a